src/main/java/com/google/sps/servlets/Recipe.java

```diff
@@ -24,12 +24,13 @@
   private String name;
   private String description;
   private String difficulty;
-  private String cookTime;
+  private Integer cookTime;
```

Comment:
Cook time and prep time are now just one parameter so you do not need them both.